KP5-Topic2/KP1-Topic2/target/surefire-reports/TEST-com.aguirre.KP1_Topic2.AppTest.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files